src/commons/achievement/AchievementView.tsx

```diff
@@ -59,7 +75,13 @@ function AchievementView(props: AchievementViewProps) {
           ))}
         </span>
       </div>
-      <AchievementViewGoal goals={goals} />
+
+      {status === AchievementStatus.COMPLETED && assessment && assessment.type === 'Missions' && (
```

Comment:
The frontend should not be hardcoded with "Missions". Perhaps you are looking for `isManuallyGraded` in assessment configs? We have this boolean column in the backend db to track assessment types that are manually graded. This assessment config is fetched when we load a new course/ refresh the page

Reply:
Would this entail going into the redux store again to find the assessment config that matches the assessment that is currently selected? Since the assessment config is decoupled from the assessment object